cmd/kcp/kcp.go

```diff
@@ -78,8 +78,8 @@ func main() {
 		}
 	}
 
-	serverOptions := options.NewOptions(rootDir)
-	serverOptions.Server.GenericControlPlane.Logs.Verbosity = logsapiv1.VerbosityLevel(2)
+	kcpOptions := options.NewOptions(rootDir)
```

Comment:
Not sure how this is cleaner :) maybe `kcpServerOptions` - to implied this is still server options, not some other kcp related options?

Reply:
I found `serverOptions.Server` to be confusing to read, thinking that this struct here holds all the options you can pass to `kcp`, hence `kcpOptions`, of which the server part is only a part.

But I'm also not super married to this, if you want I can revert it.